src/modules/acbs/acbs.service.ts

```diff
@@ -0,0 +1,41 @@
+import { HttpService } from '@nestjs/axios';
+import { Inject, Injectable } from '@nestjs/common';
+import { ConfigType } from '@nestjs/config';
+import AcbsConfig from '@ukef/config/acbs.config';
+import { AxiosError } from 'axios';
+import { catchError, lastValueFrom } from 'rxjs';
+
+import { AcbsPartyExternalRatingsResponseDto } from './dto/acbs-party-external-ratings-response.dto';
+import { AcbsException } from './exception/acbs.exception';
+import { AcbsResourceNotFoundException } from './exception/acbs-resource-not-found.exception';
+
+@Injectable()
+export class AcbsService {
+  constructor(
+    @Inject(AcbsConfig.KEY)
+    private readonly config: Pick<ConfigType<typeof AcbsConfig>, 'baseUrl'>,
+    private readonly httpService: HttpService,
+  ) {}
+
+  async getExternalRatingsForParty(partyIdentifier: string, authToken: string): Promise<AcbsPartyExternalRatingsResponseDto> {
```

Comment:
shouldn't this service sit in `party-external-rating` folder?

Reply:
The answer to this will depend on what we decide to do about the module structure as we're discussing [here](https://github.com/UK-Export-Finance/tfs-api/pull/11#discussion_r1124871205) :)

Reply:
@IgnatG as mentioned in https://github.com/UK-Export-Finance/tfs-api/pull/11#discussion_r1127923764, I've keep this service in the `acbs` module

Going forward, I'd expect the `acbs` module to be the home of all code for interacting with ACBS, essentially building up a client library for the ACBS API